api_v3/lib/types/entry/KalturaLiveStreamEntry.php

```diff
@@ -221,6 +231,8 @@ public function validateForInsert($propertiesToSkip = array())
 			$this->validatePropertyNotNull("encodingIP1");
 			$this->validatePropertyNotNull("encodingIP2");
 		}
+		$this->validatePropertyNumeric("segmentDuration");
+		$this->validatePropertyMinMaxValue("segmentDuration",MIN_ALLOWED_SEGMENT_DURATION_MILLISECONDS ,MAX_ALLOWED_SEGMENT_DURATION_MILLISECONDS );
```

Comment:
This will cause notices in the log.
Please use self:: when accessing class constants. 
For example self:: MIN_ALLOWED_SEGMENT_DURATION_MILLISECONDS